cfgov/ask_cfpb/scripts/answer_to_answerpage_migration.py

```diff
@@ -60,8 +72,10 @@ def run():
             for related_question in page.answer_base.related_questions.all():
                 page.related_questions.add(related_question.english_page)
 
-        if page.status_string != 'draft':
-            # We need to publish for the answer ID, next step, and search tags
-            page.save_revision(user=user).publish()
-        else:
+        # Save & unpublish the draft page
```

Comment:
Might be simpler to publish all pages to make sure revisions are in order, then unpublish drafts.

```python
        page.save_revision(user=user).publish()
        if draft:
            page.unpublish()
```

Reply:
Good catch!